README.md

```diff
@@ -20,15 +20,192 @@ tasks specified in the Gruntfile.
 
 #### Prerequisites
 
-- Java
-- NodeJS (12 or higher)
-- Grunt
-- Android SDK
-
-For a more complete description of how to use this repository, please see our
-user documentation at:
-
-[https://docs.odk-x.org/app-designer-intro/](https://docs.odk-x.org/app-designer-intro/)
+- Java -> Java is required by the Android SDK (installation is described below).
+- Chrome -> [Google's Chrome browser](https://www.google.com/intl/en/chrome/browser/desktop/index.html). 
+- NodeJS -> a framework for easily building fast, scalable applications (installation is described below).
+- Grunt -> a task-based scripting environment (installation is described below).
+- Android SDK -> the software development kit for Android devices (installation is described below).
+
+##### Java
+Make sure Java 8 or higher is installed on the computer you plan to use.  If it is not, [download and install it](https://java.com/en/download/). If you are using MacOSX, it may require special care and attention.See [MacOSX Java install](https://docs.oracle.com/javase/8/docs/technotes/guides/install/mac_jdk.html) and [MacOSX Java FAQ](https://docs.oracle.com/javase/8/docs/technotes/guides/install/mac_install_faq.html).
+
+##### NodeJS
+You must use Version 12 or higher. To avoid directory path problems on Windows, we require npm version 6.9 or higher (generally npm will be bundled with NodeJS installer). Follow the [instructions to install NodeJS](https://nodejs.org/en/).
+
+###### For Windows
+When installing on Windows you can use an automated **NodeJS** installer that uses **Chocolatey**. If you chose not to let the installer use **Chocolatey** to install a bunch of packages after installing **NodeJS**, you will need to ensure the location of the `npm` folder is added to the *PATH* variable of your system. If it is not, subsequent calls to access grunt will fail. For example: `C:\Users\[username]\AppData\Roaming\npm`. For instructions on modifying *PATH*, see the section at the bottom of this page called Add adb to your *PATH* For Windows. Instead of navigating to the location of Android SDK, navigate to the location of the `npm` folder.
+
```

Comment:
In windows you can open the "cmd" or "PowerShell" and check to see if npm is working by typing npm --version like suggested in Mac/Unix

Reply:
@wbrunette Sure, working on it!